docs/ops/image/NV12toBGR_8.md

```diff
@@ -0,0 +1,82 @@
+## NV12toBGR <a name="NV12toBGR"></a> {#openvino_docs_ops_image_NV12toBGR_8}
+
+**Versioned name**: *NV12toBGR-8*
+
+**Category**: *Image processing*
+
+**Short description**: *NV12toBGR* performs image conversion from NV12 format to BGR.
+
+**Detailed description**:
+
+Similar to *NV12toRGB* but output channels for each pixel are reversed so that: first channel is `blue`, second one is `green`, last one is `red`.  See detailed conversion formulas at [NV12toRGB](NV12toRGB_8.md)
+
+**Inputs:**
+
+Same as specified for [NV12toRGB](NV12toRGB_8.md) operation.
```

Comment:
does this link work?

Reply:
I tried it locally, it worked well. In Azure it seems that upload of generated docs is disabled
![image](https://user-images.githubusercontent.com/79140263/134522631-c02b0f4e-7187-4510-8c5a-b3c2d2eb49e1.png)